src/features/offer/utils.ts

```diff
@@ -122,3 +123,66 @@ export const getAllPiersIdentifiers = (data: OFFER_PAGE | undefined): PierTab[]
     return [...acc, pierTab];
   }, []);
 };
+
+type Map = {
+  id: string;
+  url: string;
+};
+
+export const getHarbor = (data: OFFER_PAGE | undefined): HarborCardProps | null => {
```

Comment:
Would it make sense to write a basic unit test for this?

Reply:
Sure, and I'll throw in tests for the rest of the utils too